### PR TITLE
Improve the benchmark comparison script

### PR DESCRIPTION
**Pull request by irevoire**
This modification allow us to compare more than 2 benchmarks or to only print the results of one benchmark

### CODE REVIEW

**Review by curquiza — COMMENTED** [review]
Should we change the README section? https://github.com/meilisearch/milli/tree/main/benchmarks#comparison-between-benchmarks

```
The benchmark reports we push are generated with critcmp. Thus, we use critcmp to generate comparison results between 2 benchmarks.
```
-> still 2?  Seems like more are possible

**Comment by irevoire** [comment]
@curquiza, yes, you're right! I updated the README.
I was also looking for a new name for the script but found nothing really good :unamused:

File: benchmarks/README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ MILLI_BENCH_DATASETS_PATH=~/datasets cargo bench --bench songs # the code is com
 
 ## Comparison between benchmarks
 
-The benchmark reports we push are generated with `critcmp`. Thus, we use `critcmp` to generate comparison results between 2 benchmarks.
+The benchmark reports we push are generated with `critcmp`. Thus, we use `critcmp` to show the result of a benchmarks, or compare results between multiple benchmarks.
```
Comment (curquiza) [inline benchmarks/README.md:57]:
```suggestion
The benchmark reports we push are generated with `critcmp`. Thus, we use `critcmp` to show the result of a benchmark, or compare results between multiple benchmarks.
```

**Review by curquiza — DISMISSED** [review]
Really clear :)

**Review by curquiza — APPROVED** [review]
Top moumoute

**Comment by irevoire** [comment]
bors merge

**Pull request merged**